quixstreams/dataframe/series.py

```diff
@@ -185,8 +185,33 @@ def test(self, value: Any, ctx: Optional[MessageContext] = None) -> Any:
         composed = self.compose()
         return context.run(composed, value)
 
+    @overload
     def _operation(
-        self, other: Union[Self, object], operator_: Callable[[object, object], object]
+        self,
+        other: Union[str, int],
+        operator_: Callable[
+            [Union[Mapping, Self], Union[str, int]], Union[Self, object]
+        ],
+    ) -> Self:
+        # getitem
+        ...
+
+    @overload
+    def _operation(
+        self,
+        other: Union[Self, object],
+        operator_: Callable[[Union[Container, Self], object], bool],
+    ) -> Self:
+        # contains
+        ...
+
+    def _operation(
+        self,
+        other: Union[Self, str, int, object],
```

Comment:
As we discussed, these are a bit ugly based on the huge range of functions we're using for it.

I think we can maybe make a separate MR if we find they are causing issues on the user end, otherwise I'm not sure it's worth the time for us to mess with.

I went ahead and dropped the overloads since they weren't really that helpful.